sympy/core/relational.py

```diff
@@ -790,16 +699,17 @@ def __new__(cls, lhs, rhs, **options):
             # nor a subclass was able to reduce to boolean or raise an
             # exception).  In that case, it must call us with
             # `evaluate=False` to prevent infinite recursion.
-            r = cls._eval_relation(lhs, rhs)
-            if r is not None:
-                return r
-            # Note: not sure r could be None, perhaps we never take this
-            # path?  In principle, could use this to shortcut out if a
-            # class realizes the inequality cannot be evaluated further.
+            return cls._eval_relation(lhs, rhs, **options)
 
         # make a "non-evaluated" Expr for the inequality
         return Relational.__new__(cls, lhs, rhs, **options)
 
+    @classmethod
+    def _eval_relation(cls, lhs, rhs, **options):
+        return cls._eval_fuzzy_relation(lhs, rhs)
```

Comment:
I'm not sure I understand the purpose of this method. Is it there for backward compatibility?